devsetup/edpm/edpm-play.yaml

```diff
@@ -1,242 +0,0 @@
-                  edpm_nova_compute_placement_project_name: service
-                  edpm_nova_compute_placement_region_name: regionOne
-                  edpm_nova_compute_placement_user_domain_name: Default
-                  edpm_nova_compute_placement_username: placement
-                  edpm_nova_compute_placement_valid_interfaces: internal
-                  edpm_nova_compute_service_user_auth_type: password
-                  edpm_nova_compute_service_user_auth_url: http://keystone-internal.openstack.svc:5000
-                  edpm_nova_compute_service_user_password: 12345678
-                  edpm_nova_compute_service_user_project_domain_name: Default
-                  edpm_nova_compute_service_user_project_name: service
-                  edpm_nova_compute_service_user_region_name: regionOne
-                  edpm_nova_compute_service_user_send_service_user_token: true
-                  edpm_nova_compute_service_user_user_domain_name: Default
-                  edpm_nova_compute_service_user_username: nova
-                  edpm_nova_compute_vnc_novncproxy_base_url: http://192.168.24.3:6080
-                  edpm_ovn_dbs:
-                  - _EDPM_OVN_DBS_
-            vars:
-                registry_name: "quay.io"
-                registry_namespace: "tripleozedcentos9"
-                image_tag: "current-tripleo"
-                edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
-                edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
-                edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
-                edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
-                edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
-                edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
-
-                gather_facts: false
-                enable_debug: false
-                # edpm firewall, change the allowed CIDR if needed
-                edpm_sshd_configure_firewall: true
-                edpm_sshd_allowed_ranges: _EDPM_SSHD_ALLOWED_RANGES_
-                # SELinux module
-                edpm_selinux_mode: enforcing
-                undercloud_hosts_entries: []
-                # edpm_hosts_entries role
-                extra_hosts_entries:
-                - 172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc
-                - 172.17.0.85 rabbitmq.openstack.svc
-                - 172.17.0.86 rabbitmq-cell1.openstack.svc
```

Comment:
This is also needed in the inventory as otherwise the nova-compute service cannot resolve `rabbitmq-cell1.openstack.svc:5672`